src/Components/VerticalSplitter.lua

```diff
@@ -56,6 +58,22 @@ function VerticalSplitter:init()
 	end
 end
 
+function VerticalSplitter:updateMouseIcon()
+	if self.state.hovering or self.state.dragging then
+		pluginMouse.Icon = "rbxasset://SystemCursors/SplitEW"
+	else
+		pluginMouse.Icon = "rbxasset://SystemCursors/Arrow"
```

Comment:
Can we instead do something like `Icon = ""` so it resets?

Reply:
Thanks, wasn't aware of this behavior